sixtyfps_runtime/corelib/Cargo.toml

```
@@ -27,6 +27,8 @@ default = ["std"]
 # from a single core, and not in a interrupt or signal handler.
 unsafe_single_core = []
 
+atomic_polyfill = ["triomphe/atomic-polyfill", "atomic-polyfill"]
```

```suggestion
atomic-polyfill = ["triomphe/atomic-polyfill"]
```
i think that should be enough.  having an underscore is confusing.

But i think this doesn't need to be an optional dependency.  But can't be merged if the triomphe patch is not released.

(also, maybe we could use SharedVector within SharedString, instead of triomphe)